docs/guides/formulasC2/build-template.md

```diff
@@ -0,0 +1,260 @@
+
+    {{site.console}} shows the first trigger node and the step that you just created in the formula visualization. Note that the step nodes include **onSuccess** <img src="img/btn-onSuccess.png" alt="OnSuccess" class="inlineImage"> and **onFailure** <img src="img/btn-onFailure.png" alt="OnFailure" class="inlineImage"> buttons.
+    ![Trigger](img/viz-trigger-step.png)
+
+7. To add another step, either click **onSuccess** <img src="img/btn-onSuccess.png" alt="OnSuccess" class="inlineImage"> or **onFailure** <img src="img/btn-onFailure.png" alt="OnFailure" class="inlineImage">, depending on the previous step.
+8. Continue adding steps until you complete the formula template.
+9. Add a description by clicking Edit. See [Add a Description to a Formula Template](#add-a-description-to-a-formula-template).
+8. After you complete the formula, [test it](#test-formula-templates).
+
+{% include note.html content="Your formula template is saved in the background as you add each component." %}
+
+## Test Formula Templates
+
+You can test a formula template as you build it. When you test a formula template, you can either use an existing formula instance or create a new formula instance.
+
+{% include tip.html content=" If you plan to use the formula instance that you create while testing only for testing, consider appending the name test to it. " %}
+
+To test a formula template:
+
+1. At the top of the formula visualization, click **Setup Test**.
+![Setup Test](img/set-up-test.png)
+2. Click **Select Instance**.
+3. Either choose an existing formula instance (and skip down to the Select Trigger step) or click **Add New Instance**.
+  4. Enter a name for the new formula instance.
+  5. For each variable, click the variable and add an element instance or enter a value.
+  6. Click **Create Instance**.
+  7. Select the instance that you just created.
+8. Click **Select Trigger**.
+9. Provide trigger information, depending on the type of trigger.
+
+## Create Variables
+
+You can create two types of variables: Element Instance variables that will be replaced by element instances and Value variables that will be replaced by values entered at the formula instance. When you create a variable, note the Formula Step Variable Name which is how you refer to the variable in the formula context. See [Formula Variables](reference.html/#formula-variables) for more about variables in the formula context.
+
+To create a formula variable:
+
+1. Open the formula template. On the Formulas page, hover over the Formula Card, and then click **Open**.
+5. Click **Variables**.
+![Variables](img/variables.png)
+2. Select the type of variable to create.
+3. Enter a name for the variable. You can include spaces, but spaces are removed to create the Formula Step Variable Name, which is how the variable is referred to in the formula context.
```

Comment:
The spaces are removed when creating the formula step `key`, not the `name`.  That remains un-touched, I believe?